documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -145,12 +145,12 @@ apiVersion: forklift.konveyor.io/v1beta1
 kind: NetworkMap
 metadata:
   name: <network_map>
-  namespace: {namespace}
+  namespace: <namespace>
 spec:
   map:
     - destination:
-        name: <pod>
-        namespace: {namespace}
+        name: <network_name>
+        namespace: <namespace>
```

Comment:
@RichardHoch and let's drop this setting of the namespace as it's not needed please

Reply:
Yeah, i was wondering about that. Done.